src/main/java/seedu/address/logic/command/sort/SortAccommodationCommand.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.logic.command.sort;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Comparator;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import seedu.address.logic.command.CommandResult;
+import seedu.address.logic.command.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.accommodation.Accommodation;
+import seedu.address.model.accommodation.UniqueAccommodationList;
+import seedu.address.model.travelplan.AccommodationList;
+import seedu.address.model.travelplan.TravelPlan;
+
+
+public class SortAccommodationCommand extends SortCommand {
+    public static final String COMMAND_WORD = "sort accommodation";
+
+    public static final String MESSAGE_USAGE =
+            "sort accommodation: Sorts the list of accommodation in a travel plan by the keyword input by the user.\n"
+                    + "Parameters: KEYWORD (COST/DATE)\n";
```

Comment:
Should the cost and date be lowercase? maybe others will misinterpret it as uppercase and type that instead

Reply:
ok